data/card_catalogue/upgradeable-1.json

```diff
@@ -331,9 +331,9 @@
     "details": {
       "rules": {
         "active": "Restore a visible character to maximum health. Their maximum health permanently increases by 10 and they grow 3 feet taller.",
-        "passive": "You are 3 feet taller."
```

Comment:
This passive doesn't feel very impactful given what the card does. Since we're low on text space for this card, I don't think we need it